dccrg.hpp

```diff
@@ -7913,7 +7860,8 @@ template <
 
 			// Base neighbourhood is symmetric, so just construct symmetric complement.
 			// this->neighbors_to[item.first] = this->find_neighbors_to(item.first, this->neighborhood_to);
-			this->neighbors_to[item.first] = this->fill_neighbors_to(item.first, this->neighborhood_to);
+			// this->neighbors_to[item.first] = this->fill_neighbors_to(item.first);
+			this->neighbors_to[item.first] = this->neighbors_of[item.first];
```

Comment:
This makes me wonder whether we can just throw neighbors_to overboard altogether now?
(Not the user data ones, of course, but at least the base neighborhood, that we know is symmetric.)

Anyway, that should be its own PR, I suppose

Reply:
We could, yes. The challenging bit will be to get all the half-rotten debug code to also understand that change... :)